src/Analysis/GWASResults/Views/Input/AttritionTable/AttrtitionTable.jsx

```diff
@@ -4,12 +4,61 @@ import PropTypes from 'prop-types';
 import './AttritionTable.css';
 
 const { Panel } = Collapse;
+
+const defaultHareGroups =  ['Non-Hispanic Black','Non-Hispanic Asian','Non-Hispanic White','Hispanic']
+
 const AttritionTable = ({ tableData, title }) => {
+  const getHareGroups = (conceptBreakdownArray) => {
+    let groupNames = conceptBreakdownArray.map(item => {
+      return { concept_value_name: item.concept_value_name };
+    });
+    return groupNames;
+  };
+
+  const displayHareGroupHeaders = (hareGroupNames) => {
+    let i = 0;
+    let buffer = [];
+
+    while (i < hareGroupNames.length)
+    {
+      let groupName = hareGroupNames[i]['concept_value_name'];
+      groupName = groupName.charAt(0).toUpperCase() + groupName.slice(1);
+
+      if(i == 0)
+      {
+        buffer.push(<th className='attrition-table--w15 attrition-table--leftpad'>{groupName}</th>);
+      }
+      else
+      {
+        buffer.push(<th className='attrition-table--w15'>{groupName}</th>);
+      }
+      i++;
+    }
+
+    return buffer;
+  }
+
+  const displayGroupBreakDowns = (hareGroupNames, row) => {
+    let i = 0;
+    let buffer = [];
+
+    while (i < hareGroupNames.length)
+    {
+      let groupName = hareGroupNames[i]["concept_value_name"];
+      let count = getBreakDownForGroup(groupName, row?.concept_breakdown)
```

Comment:
It looks like displayGroupBreakDowns calls getBreakDownForGroup with the possibly undefined .concept_breakdown : ```getBreakDownForGroup(groupName, row?.concept_breakdown);```
However getBreakDownForGroup  operates on this assuming it is an array on ln 50. 
You might want to provide a fallback on this if it is possible that it is not an array.